pyproject.toml

```diff
@@ -158,12 +162,21 @@ ignore = [
     # DoNotAssignLambda
     "E731",
 ]
+exclude = ["*.ipynb"]
+
+[tool.ruff.lint.pydocstyle]
+# Use Google-style docstrings.
+convention = "numpy"
 
 [tool.ruff.lint.per-file-ignores]
 # Ignore `E402` and `F401` (unusued imports) in all `__init__.py` files
 "__init__.py" = ["E402", "F401"]
-# Ignore assert statements in tests
-"tests/*" = ["S101"]
+"tests/*" = [
+    # Allow assert statements in tests
+    "S101",
+    # Disable docstring checks in tests
+    "D",
```

Comment:
Would it make sense to address this eventually? I can understand how it might be a big task to account for this, but feel it may be healthy to document tests at a certain level (otherwise we may set the precedent that we don't need to explain tests for others).